src/lib/PostgresMetaFunctions.ts

```diff
@@ -196,6 +212,84 @@ export default class PostgresMetaFunctions {
     }
     return { data: func!, error: null }
   }
+
+  private generateCreateFunctionSql(
+    {
+      name,
+      schema,
+      args,
+      argument_types,
+      definition,
+      return_type,
+      language,
+      behavior,
+      security_definer,
+      config_params,
+    }: Partial<Omit<FunctionInputs, 'config_params'> & PostgresFunction>,
+    { replace = false, terminateCommand = true } = {}
+  ): string {
+    return `
+      CREATE ${replace ? 'OR REPLACE' : ''} FUNCTION ${ident(schema!)}.${ident(name!)}(${
+      argument_types || args?.join(', ') || ''
+    })
+      RETURNS ${return_type}
+      AS ${literal(definition)}
+      LANGUAGE ${language}
+      ${behavior}
+      ${security_definer ? 'SECURITY DEFINER' : 'SECURITY INVOKER'}
+      ${
+        config_params
+          ? Object.entries(config_params)
+              .map(
+                ([param, value]) =>
+                  `SET ${param} ${value[0] === 'FROM CURRENT' ? 'FROM CURRENT' : 'TO ' + value}`
+              )
+              .join('\n')
+          : ''
+      }
+      RETURNS NULL ON NULL INPUT
```

Comment:
The default is `CALLED ON NULL INPUT` right? Any reason for this one?

Reply:
@soedirgo i'll make the update to the default